src/views/BridgeTransactionsHistory.vue

```diff
@@ -31,9 +31,14 @@
         </s-form-item>
         <div class="history-items">
           <template v-if="hasHistory">
-            <div v-for="item in filteredHistory.slice((currentPage - 1) * 10, currentPage * 10)" :key="item.id" class="history-item" :data-id="item.id || item.hash || item.ethereumHash" @click="showHistory(item.id || item.hash || item.ethereumHash)">
+            <div
+              class="history-item"
+              v-for="item in filteredHistory.slice((currentPage - 1) * 10, currentPage * 10)"
```

Comment:
10 is pageSize right? May we add this data variable?

Reply:
yes sure)))